api/src/main/java/org/openmrs/serialization/SimpleXStreamSerializer.java

```diff
@@ -100,4 +107,28 @@ public String serialize(Object o) throws SerializationException {
 			throw new SerializationException("Unable to deserialize class: " + clazz.getName(), e);
 		}
 	}
+	
+	/**
+	 * An instance of this converter needs to be registered with a higher priority than the rest so
+	 * that it's called early in the converter chain. This way, we can make sure we never get to
+	 * xstream's DynamicProxyConverter that can deserialize proxies which can be a channel for
+	 * attacks.
```

Comment:
could you provide a link to something that documents/describes why we are doing this?